operator_console/tests/test_main.py

```
@@ -1840,7 +1849,30 @@ def media_file_validate(self, *, folder_path: str) -> dict[str, object]:
 
 
 class _FakeAdminServices:
+    _editable = {
+        "video_thumbnails_enabled": ("ui", "bool"),
+        "canonical_read_cache_enabled": ("performance", "bool"),
+        "canonical_read_cache_ttl_seconds": ("performance", "float"),
+    }
```

Ruff (0.15.9)

[warning] Mutable default value for class attribute

(RUF012)